src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.tsx

```diff
@@ -102,30 +94,30 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
 
     const {redBrickDateIndicator, redBrickTimeIndicator} = useMemo(
         () => ({
-            redBrickDateIndicator: customDateError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : null,
-            redBrickTimeIndicator: customTimeError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : null,
+            redBrickDateIndicator: customDateError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            redBrickTimeIndicator: customTimeError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
         }),
         [customTimeError, customDateError],
     );
 
     const onSubmit = () => {
-        const {dateError, timeError} = validateCustomDate();
+        const {dateValidationErrorKey: dateError, timeValidationErrorKey: timeError} = validateCustomDate();
```

Comment:
This is a big change in logic, this wasn't working before?

Reply:
Is it? I thought that I was just renaming the fields 😅

Reply:
Renaming works the other way around:
```ts
const {dateError: dateValidationErrorKey, timeError: timeValidationErrorKey} = validateCustomDate();
```

Looking at the previous code (mostly `validateCustomDate`) I assume this was a mistake in the logic.

Reply:
I think you can fix the other way around:
```ts
    const validate = () => {
        const {dateValidationErrorKey, timeValidationErrorKey} = ValidationUtils.validateDateTimeIsAtLeastOneMinuteInFuture(data);

        setCustomDateError(dateValidationErrorKey);
        setCustomTimeError(timeValidationErrorKey);

        return {
            dateError: dateValidationErrorKey,
            timeValidationErrorKey: timeValidationErrorKey,
        };
    };
```